### PR TITLE
Basic UI 

### PR DESCRIPTION
**Pull request by srishti-maurya**
## Added
- Landing Page
- Pages (Login, Logout)
- Component (Nav and SideBar)
- Router 

## Screenshot for reference
screencapture-localhost-3000-2022-03-28-17_21_54
screencapture-localhost-3000-home-2022-03-28-17_22_16
screencapture-localhost-3000-login-2022-03-28-17_22_27
screencapture-localhost-3000-logout-2022-03-28-17_22_37

### CODE REVIEW

**Review by MarkVed17 — APPROVED** [review]
I like the naming conventions followed. Improves code-readability. Also, good work with the UI. You may proceed.

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+      <Routes>
+        <Route path="/" element={<LandingPage />} />
+        <Route path="home" element={<Home />} />
+        <Route path="archive" element={<Archive />} />
+        <Route path="login" element={<Login />} />
+        <Route path="logout" element={<Logout />} />
+        <Route path="signup" element={<Signup />} />
+        <Route path="labels" element={<Labels />} />
+        <Route path="profile" element={<Profile />} />
+        <Route path="trash" element={<Trash />} />
```
Comment (Chakravarthi0) [inline src/App.js:30]: you can create a separate component for routes

File: src/frontend/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export { Home } from "./Home/Home.jsx";
+export { Archive } from "./Archive/Archive.jsx";
+export { Login } from "./Auth/Login.jsx";
+export { Signup } from "./Auth/Signup.jsx";
+export { Logout } from "./Auth/Logout.jsx";
+export { Labels } from "./Labels/Labels.jsx";
+export { LandingPage } from "./LandingPage/LandingPage.jsx";
+export { Profile } from "./Profile/Profile.jsx";
+export { Trash } from "./Trash/Trash.jsx";
```
Comment (Chakravarthi0) [inline src/frontend/pages/index.js:9]: I like the way you are using barrel export

File: src/frontend/styles/styles.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+.cursor-pointer {
+  cursor: pointer;
+}
+
+/* landing page */
+
+.hero-img {
+  width: 700px;
```
Comment (Chakravarthi0) [inline src/frontend/styles/styles.css:90]: You can use rem here

File: src/index.js [hunk 4]
```diff
@@ -2,13 +2,16 @@ import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App";
 import { makeServer } from "./server";
+import { BrowserRouter } from "react-router-dom";
```
Comment (Chakravarthi0) [inline src/index.js:5]: Just a suggestion: you can rename it to router.

**Review by Chakravarthi0 — APPROVED** [review]

**Pull request merged**